### PR TITLE
Re-added correct schemas to health path.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides_backend/action/mixins/singular_action_mixin.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from ...shared.exceptions import ActionException
 from ...shared.schema import schema_version
+from ..action import Action
 from ..util.typing import ActionPayload
 
 singular_schema = fastjsonschema.compile(
```
Comment (jsangmeister) [inline openslides_backend/action/mixins/singular_action_mixin.py:8]: This schema is obsolet now, you can just use the `action_payload_schema`

File: openslides_backend/action/action_handler.py [hunk 2]
```diff
@@ -17,11 +17,20 @@
     Payload,
 )
 
+action_payload_schema = {
+    "$schema": schema_version,
+    "title": "Action payload",
+    "type": "array",
+    "items": {"type": "object"},
+    "minItems": 1,
+    "uniqueItems": True,
```
Comment (jsangmeister) [inline openslides_backend/action/action_handler.py:26]: This is not needed. I removed it in a previous PR from the other schemas. It should be possible to just give an empty list of payloads - then nothing happens. Also, the items don't have to be unique. Imagine a case where you want for some reason to create the same object (with the same content) multiple times. It doesn't matter if this makes sense but we should not forbid it.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/mixins/singular_action_mixin.py (hunk 1 above)
Comment (normanjaeckel) [reply openslides_backend/action/mixins/singular_action_mixin.py:8]: I do not understand you objection. The SingularActionMixin ensures that one can only send one item inside specific action payloads.

Comment (normanjaeckel) [reply openslides_backend/action/mixins/singular_action_mixin.py:8]: Please open new PR if something is still open.

File: openslides_backend/action/action_handler.py (hunk 2 above)
Comment (normanjaeckel) [reply openslides_backend/action/action_handler.py]: Fixed.

**Pull request merged**